test/unit/model/v2/test_pre_validate.py

```diff
@@ -18,14 +18,14 @@
 #  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 #  DEALINGS IN THE SOFTWARE.
 
-import pytest
-
 from molecule.model import schema_v2
 
+# test data dict used to generate tests:
+# key is the test's string id, used in pytest verbose output
+# value is a tuple of (input_molecule_yml_section_data, expected_schema_errors)
+_pre_validate_test_data = {}
 
-@pytest.fixture
-def _model_platforms_docker_section_data():
```

Comment:
What deprecation is this linked to?

Reply:
Sorry about that, I meant to include refs to each specific deprecation being addressed in each commit. I'll get those added next week. For now, though, this was related to the test functions parametrizing `_stream` with the fixture-decorated function itself, not the resolved fixture value, resulting in the fixture function having to be called (e.g. `_stream()`), which is deprecated in pytest 4+: https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly